data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -215,8 +234,8 @@ while true; do
         sleep 10
       done
 
-      echo "Stopping provisioning services..."
-      podman stop ironic-api ironic-conductor ironic-inspector ironic-ramdisk-logs $dnsmasq_container_name httpd mariadb
+      echo "PRETENDING to Stopping provisioning services..."
+      #podman stop ironic-api ironic-conductor ironic-inspector ironic-ramdisk-logs $dnsmasq_container_name httpd mariadb
```

Comment:
Presumably this is disabled for debugging, but when re-enabled we can probably stop the image-customization container here too?